Android/Fundamentals/drawables-ii/create-a-state-list.md

```diff
@@ -0,0 +1,155 @@
+# create-state-list
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: how to
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+Step 1: Create a button 
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android=
+"http://schemas.android.com/
+apk/res/android">
+<item
+android:state_focused="true"
+android:state_enabled="true"
+android:drawable=
+"@drawable/button_focused" />
+<item
+android:state_enabled="true"
+android:drawable=
+"@drawable/button_normal" />
+</selector>
+```
+Step 2: Create the drawables that are listed in the button.
+Button Normal
```

Comment:
list that these are different button states- you might show the button and then, say "to make the different states, this line is the one you need to change" and then only show the lines that are actually different.